.dvc/config

```diff
@@ -0,0 +1,6 @@
+[core]
+    remote = localremote
+['remote "gdrive-remote"']
+    url = gdrive://1FDl4yPwA_v0BBjvbeHNQn56vAiYdOQyn
+['remote "localremote"']
+    url = ../../dvcstore
```

Comment:
Didn't want to change the `.gitignore` to ignore a separate folder so that's why I kept it separate. I'm ok with either choice. Feel free to change this if you want